layer-atlas/src/main/java/com/layer/ui/avatar/AvatarView.java

```
@@ -122,9 +122,11 @@ public void setParticipants(Identity... participants) {
      * Should be called from UI thread.
      */
     public void setParticipants(Set<Identity> participants) {
-        mParticipants.clear();
```

If you receive a null argument, throw a `NullPointerException` in case the underlying method won't accept it. Don't silently swallow the error.